src/Terminus.php

```diff
@@ -201,6 +203,8 @@ private function addBuiltInCommandsAndHooks()
     private function configureContainer(ContainerInterface $container)
     {
         // Add the services.
+        $container->add(Client::class);
+        $container->add(HttpRequest::class);
```

Comment:
Why are you adding `Client` and `HttpRequest` to the container? These classes shouldn't ever need to use dependency injection, and since there's no generic interface, injecting them into some other class is dubious as well.

It would be much better if you just used `new` to instantiate these. There are a number of other classes that could, and should, be removed from the container. This pattern is leading to using `ContainerAwareInterface` more often than necessary. Only classes that really need a reference to the container should use it.